lib/core/config-schema.js

```
@@ -0,0 +1,204 @@
+      is: 'sqlite',
+      then: Joi.optional(),
+      otherwise: Joi.required()
+    }),
+    username: Joi.string().when('dialect', {
+      is: 'sqlite',
+      then: Joi.optional(),
+      otherwise: Joi.required()
+    }),
+    password: Joi.string().when('dialect', {
+      is: 'sqlite',
+      then: Joi.optional(),
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)